test/grdimage/rounding.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Failing script while fixing https://forum.generic-mapping-tools.org/t/error-pygmt-gmtcliberror-module-grdimage-failed-with-status-code-78/829
+# Now reported as issue https://github.com/GenericMappingTools/gmt/pull/4130
+
+# Ensure the two tiles are downloaded first:
+gmt grdcut -R3:57/4:18/44:00/44:15 @earth_relief_03s -G/dev/null
```

Comment:
No. I didn't mean this. It should be done in the CI script:

https://github.com/GenericMappingTools/gmt/blob/66f5bdce05d4e34ecc371633da95d358f517dd57/ci/azure-pipelines-mac.yml#L80-L90

Reply:
OK, I agree. Should I or you move that code to the .yml file then?

Reply:
I'll do that.